src/kbmod/filters/known_object_filters.py

```diff
@@ -34,7 +34,6 @@ def __init__(
         matcher_name,
         sep_thresh=1.0,
         time_thresh_s=600.0,
-        match_min_obs=5,
         match_obs_ratio=0.5,
```

Comment:
I don't think you need this parameter any longer.

Reply:
Done